assets/js/util/tracking/index.js

```diff
@@ -54,14 +54,14 @@ const {
  */
 function toggleTracking( activeStatus ) {
 	if ( !! activeStatus ) {
-		enableTracking();
+		enableTracking.initializeSnippet();
 	} else {
 		disableTracking();
 	}
 }
 
 // Bootstrap on import if tracking is allowed.
-if ( true === trackingAllowed ) {
+if ( true === isSiteKitScreen ) {
 	toggleTracking( isTrackingEnabled() );
```

Comment:
This is where we initialize the snippet now. `toggleTracking( true )` would do the same here, but it's a bit unnecessarily confusing.

```suggestion
if ( true === isSiteKitScreen && isTrackingEnabled() ) {
	initializeSnippet();
```